mp/src/game/shared/da/da_player_shared.h

```diff
@@ -231,6 +231,7 @@ class CDAPlayerShared
 	CNetworkVar( bool, m_bIsTryingUnprone );
 #endif
 
+	// > 0 apparently means getting up from slide (?)
 	CNetworkVar( float, m_flUnSlideTime );
```

Comment:
Yeah I thought I'd need to know what that variable does, but in the end I could fix the bug without knowing. I wrote down what I know in that comment.